src/coreclr/src/vm/compatibilityswitch.cpp

```diff
@@ -23,20 +23,7 @@ FCIMPL2(StringObject*, CompatibilitySwitch::GetValue, StringObject* switchNameUN
     HELPER_METHOD_FRAME_BEGIN_RET_1(name);
     CLRConfig::ConfigStringInfo info;
     info.name = name->GetBuffer();
-    if(onlyDB)
-    {
-        // for public managed apis we ignore checking in registry/config/env
```

Comment:
Is CLRConfig::IgnoreHKCU etc still used?

Reply:
I guess one can read COMPlus_xx from registry?

Reply:
> Is CLRConfig::IgnoreHKCU etc still used?

It is not. There is a lot of other unused stuff. I will leave it for separate PR.

Reply:
https://github.com/dotnet/runtime/issues/40028